src/main/java/org/isf/vaccine/gui/VaccineEdit.java

```diff
@@ -131,10 +126,10 @@ public VaccineEdit(JFrame owner, Vaccine old, boolean inserting) {
 	private void initialize() {
 		Toolkit kit = Toolkit.getDefaultToolkit();
 		Dimension screensize = kit.getScreenSize();
-		pfrmWidth = 350;
-		pfrmHeight = 250;
-		pfrmBordX = screensize.width / 2 - (pfrmWidth / 2);
-		pfrmBordY = screensize.height / 2 - (pfrmHeight / 2);
+		int pfrmWidth = 350;
+		int pfrmHeight = 250;
+		int pfrmBordX = screensize.width / 2 - (pfrmWidth / 2);
+		int pfrmBordY = screensize.height / 2 - (pfrmHeight / 2);
 		this.setBounds(pfrmBordX,pfrmBordY,pfrmWidth,pfrmHeight);
 		this.setContentPane(getJContentPane());
 		if (insert) {
```

Comment:
In many cases we just replaced all this stuff with:
```
		pack();
		setLocationRelativeTo(null);
```

`pack()` allows to double check the minimum size and to check the good implementation of the layout.
`setLocationRelativeTo(null)` allows to center the result anyhow

Can you give it a try?

Reply:
Thanks!